components/Error500Page.tsx

```diff
@@ -1,7 +1,8 @@
-import React, { Component } from "react";
+import React from "react";
 import styles from "./BasicInfo.module.css";
+import { BasicDataStatus } from "../types/index";
```

Comment:
`index` file is a special file and treated as a root of this folder. so we can take `/index` out from the path.